infra/core/search/search-services.bicep

```diff
@@ -70,4 +70,4 @@ resource search 'Microsoft.Search/searchServices@2023-11-01' = {
 output id string = search.id
 output endpoint string = 'https://${name}.search.windows.net/'
 output name string = search.name
-output principalId string = !empty(searchIdentityProvider) ? search.identity.principalId : ''
\ No newline at end of file
+output principalId string = !empty(searchIdentityProvider) ? search.identity.principalId : ''
```

Comment:
Were these EOL changes due to running `az bicep format` or from your editor?

Reply:
It's from my editor. After using the command `pre-commit run --all-files`, pre-commit hooks run on all the files in my repository.